<?xml version="1.0"?>
<launch>

    <arg name="namespace" default="chair" />
    <arg name="onboard_images_folder" default="$(find wc_perception)/onboard_images"/>
    <arg name="camera_name" default="camera" />

    <node name="wc_perception" pkg="wc_perception" type="wc_perception_node.py" output="screen">      
        <remap from="CNN_out" to="/cnn_out/predictions"/>
        <remap from="state_change" to="$(arg namespace)/state_change"/>
        <!-- JKM cluge in /webcam/image_raw topic -->
        <!--<remap from="camera" to="$(arg namespace)/image_raw"/-->
        <remap from="camera" to="$(arg camera_name)/image_raw"/>
        <param name="json_model_path" value="$(find wc_perception)/models/model_struct.json"/>
        <param name="weights_path" value="$(find wc_perception)/models/model_weights_59.h5"/>
        <param name="/onboard_images_folder" value="$(arg onboard_images_folder)"/>
        <param name="target_size" value="320, 240"/>
        <param name="crop_size" value="200,200"/>

  </node>

</launch>

